src/framework/auth/services/token/token-packer.ts

```diff
@@ -0,0 +1,44 @@
+import { Inject, Injectable } from '@angular/core';
+
+import { NB_AUTH_TOKENS } from '../../auth.options';
+import { NbAuthSimpleToken, NbAuthToken, nbCreateToken } from './token';
+
+interface NbTokenPack {
```

Comment:
Also, I don't like this naming at all. When I see *token pack* I think about a number of tokens. Not about information required to recreate token.  Please, think about it and choose more informative naming for this feature.

Reply:
Parcel.